config/config.go

```diff
@@ -37,12 +37,28 @@ type Config struct {
 	Period     time.Duration           `config:"period"`
 	Processors processors.PluginConfig `config:"processors"`
 	Fetchers   []*common.Config        `config:"fetchers"`
+
+	DataYaml struct {
+		ActivatedRules struct {
+			CISK8S []string `config:"cis_k8s"`
+		} `config:"activated_rules"`
+	} `config:"data_yaml"`
 }
 
 var DefaultConfig = Config{
 	Period: 10 * time.Second,
 }
 
+func New(cfg *common.Config) (Config, error) {
```

Comment:
Not sure where it's used (besides tests)

Reply:
Had forgotten to add those changes to the commit, fixed